monkey/monkey_island/cc/ui/src/services/configuration/masquerade.js

```diff
@@ -0,0 +1,20 @@
+const MASQUERADE = {
+  'title': 'Masquerade',
+  'type': 'object',
+  'properties': {
+    'masque_list': {
+      'title': 'Masque list',
+      'type': 'array',
+      'uniqueItems': true,
+      'items': {'type': 'string'},
+      'default': [],
+      'description': 'List of masque strings that will be appended to the agent binaries '
```

Comment:
I think we could be more descriptive about what this feature does and why it's added.